cmd/vicadmin/vicadm.go

```diff
@@ -96,8 +118,37 @@ var (
 )
 
 func init() {
-	trace.Logger.Level = log.DebugLevel
 	defer trace.End(trace.Begin(""))
+	trace.Logger.Level = log.DebugLevel
```

Comment:
Why are we doing this? If we don't want this to run as root then let's not start with root user